servicetalk-http-router-jersey/src/main/java/io/servicetalk/http/router/jersey/EndpointEnhancingRequestFilter.java

```diff
@@ -84,15 +85,17 @@
 final class EndpointEnhancingRequestFilter implements ContainerRequestFilter {
 
     private final EnhancedEndpointCache enhancedEndpointCache = new EnhancedEndpointCache();
-
-    @Context
-    private Provider<Ref<ConnectionContext>> ctxRefProvider;
-
-    @Context
-    private Provider<RouteStrategiesConfig> routeStrategiesConfigProvider;
-
-    @Context
-    private RequestScope requestScope;
+    private final Provider<Ref<ConnectionContext>> ctxRefProvider;
+    private final Provider<RouteStrategiesConfig> routeStrategiesConfigProvider;
+    private final RequestScope requestScope;
+
+    EndpointEnhancingRequestFilter(@Context final Provider<Ref<ConnectionContext>> ctxRefProvider,
```

Comment:
@ddossot - I'm assuming the constructor injection is equivalent to filed injection in this case in terms of injection and state (doesn't need to be re-assigned). wdyt?